extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+// Name: Twitch
+// ID: twitch
+// Description: Blocks that allow you to stream easier on TurboWarp.
```

Comment:
"Blocks that interact with Twitch. Unofficial." would be a better description.

Reply:
![image](https://github.com/TurboWarp/extensions/assets/140775902/5c1833ee-2e56-49d7-a37c-4c7e2c748b57)